### PR TITLE
LPD-3279 Truncated text for account name in placed and pending orders tables

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**
https://liferay.atlassian.net/browse/LPD-3279

### CODE REVIEW

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Review by riccardo-alberti — APPROVED** [review]

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/data/set/view/table/PendingCommerceOrderTableFDSView.java [hunk 1]
```diff
@@ -41,7 +42,7 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 		).add(
 			"purchaseOrderNumber", "purchase-order-number"
 		).add(
-			"accountName", "account"
+			_addFDSTableAccountField()
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/data/set/view/table/PendingCommerceOrderTableFDSView.java:45]: not sure Brian will like this method name
```suggestion
			_addAccountNameFDSTableSchemaField()
```

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/data/set/view/table/PendingCommerceOrderTableFDSView.java (hunk 1 above)
Comment (andrea-ale-sbarra) [reply modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/data/set/view/table/PendingCommerceOrderTableFDSView.java:45]: In both classes

Comment (gianmarcobrunialti) [reply modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/data/set/view/table/PendingCommerceOrderTableFDSView.java]: ✅

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**